.travis.yml

```diff
@@ -1,14 +1,20 @@
 language: c
-sudo: required
+dist: xenial
+
+addons:
+  apt:
+    packages:
+      - gfortran
+      - libopenmpi-dev
+      - libhdf5-openmpi-dev
 
 before_install:
-    - sudo apt-get clean && sudo apt-get update
-    - sudo apt-get install gcc gfortran hdf5-tools libopenmpi-dev libhdf5-openmpi-dev 
     - which gfortran
+    - gfortran --version
     - which gcc
-    - which h5dump
+    - gcc --version
 
 script:
-    - pwd
+    - set -e
```

Comment:
good catch, otherwise travis will compile with an error and still give thumbs up.

Reply:
I [learn from the best](https://github.com/EDmodel/ED2/pull/224/files#diff-354f30a63fb0907d4ad57269548329e3R27).